modules/installation-obtaining-installer.adoc

```diff
@@ -88,7 +91,7 @@ endif::ash[]
 . Navigate to the page for your installation type, download the installation program that corresponds with your host operating system and architecture, and place the file in the directory where you will store the installation configuration files.
 endif::[]
 ifdef::openshift-origin[]
-. Download installer from https://github.com/openshift/okd/releases
+. Download installer from https://github.com/openshift/okd/releases.
```

Comment:
Looks like we don't do OKD previews anymore, but I think adding a period here will break the link.

Reply:
Update complete.